koch.nim

```diff
@@ -326,9 +326,15 @@ proc boot(args: string) =
     exec "$# jsonscript --nimcache:$# $# compiler" / "nim.nim" %
       [nimi, smartNimcache, args]
 
+    # https://github.com/nim-lang/Nim/pull/16008#discussion_r524961618
```

Comment:
In the best case, this is completely unrelated code. In the worst case it shows you changed ``sameFileContent``'s behaviour.